taskweaver/cli/init.py

```diff
@@ -56,46 +56,21 @@ def init(
     if not os.path.exists(project):
         os.mkdir(project)
 
-    def get_dir(*dir: str):
-        return os.path.join(project, *dir)
-
-    dir_list = [
-        "codeinterpreter_examples",
-        "planner_examples",
-        "plugins",
-        "config",
-        "workspace",
-    ]
-    for dir in dir_list:
-        dir_path = get_dir(dir)
-        if not os.path.exists(dir_path):
-            os.mkdir(dir_path)
-
-    init_temp_dir = get_dir("init")
     import zipfile
     from pathlib import Path
 
-    tpl_dir = os.path.join(init_temp_dir, "template")
-    ext_zip_file = Path(__file__).parent / "taskweaver-ext.zip"
+    tpl_dir = os.path.join(project, "temp")
+    if not os.path.exists(tpl_dir):
+        os.mkdir(tpl_dir)
+
+    ext_zip_file = Path(__file__).parent / "taskweaver-project.zip"
     if os.path.exists(ext_zip_file):
         with zipfile.ZipFile(ext_zip_file, "r") as zip_ref:
             # Extract all files to the current directory
             zip_ref.extractall(tpl_dir)
-
-        tpl_planner_example_dir = os.path.join(tpl_dir, "taskweaver-ext", "planner_examples")
-        tpl_ci_example_dir = os.path.join(tpl_dir, "taskweaver-ext", "codeinterpreter_examples")
-        tpl_plugin_dir = os.path.join(tpl_dir, "taskweaver-ext", "plugins")
-        tpl_config_dir = os.path.join(tpl_dir, "taskweaver-ext")
-        planner_example_dir = get_dir("planner_examples")
-        ci_example_dir = get_dir("codeinterpreter_examples")
-        plugin_dir = get_dir("plugins")
-        copy_files(tpl_planner_example_dir, planner_example_dir)
-        copy_files(tpl_ci_example_dir, ci_example_dir)
-        copy_files(tpl_plugin_dir, plugin_dir)
-        copy_file(tpl_config_dir, "taskweaver_config.json", get_dir(""))
-
+        copy_files(os.path.join(tpl_dir, "project"), project)
```

Comment:
If the project dir already exists, will this overwrite the original content?

Reply:
No, have a check function named validate_empty_workspace